opentelemetry/README.md

```diff
@@ -82,11 +82,12 @@ The **OpenTelemetry** Plugin comes with a [Failover Connector](https://github.co
 `openTelemetry.logsCollector.enabled`    | Activates the standard configuration for logs | bool | `false` |
 `openTelemetry.logsCollector.kvmConfig.enabled`    | Activates the configuration for KVM logs (requires logsCollector to be enabled) | bool | `false` |
 `openTelemetry.logsCollector.cephConfig.enabled`    | Activates the configuration for Ceph logs (requires logsCollector to be enabled) | bool | `false` |
+`openTelemetry.logsCollector.failover.enabled`      | Activates the failover mechanism  for shipping logs via a fallback user if the first users' credentials have expired | bool | `false` |
```

Comment:
"First user" maybe you could highlight that it is intended to be `failover_a`? From an outsider's perspective it might be unclear which is the "first" and which is the "secondary".